classes/controller/FrontController.php

```diff
@@ -1605,9 +1625,9 @@ public function getTemplateVarShop()
             'long' => Configuration::get('PS_STORES_CENTER_LONG'),
             'lat' => Configuration::get('PS_STORES_CENTER_LAT'),
 
-            'logo' => (Configuration::get('PS_LOGO')) ? _PS_IMG_ . Configuration::get('PS_LOGO') : '',
-            'stores_icon' => (Configuration::get('PS_STORES_ICON')) ? _PS_IMG_ . Configuration::get('PS_STORES_ICON') : '',
-            'favicon' => (Configuration::get('PS_FAVICON')) ? _PS_IMG_ . Configuration::get('PS_FAVICON') : '',
+            'logo' => (Configuration::get('PS_LOGO')) ? Configuration::get('PS_LOGO') : '',
```

Comment:
Why do you need to do this? Is it because in the template `{$urls.img_ps_url}` is correctly set with the CDN url?
Because what worries me is that it changes the template vars used by all themes, so it's likely every themes will need to be updated to make their logo and favicon work again, so we'd better have a good reason for changing this

Reply:
In fact, we need to have a absolute URL : 
* In template, `$urls.img_ps_url` give me `http://127.0.0.1/PrestaShop/img/`
* While `_PS_IMG_` is just relative `/img/`